site-src/v1alpha2/contributing/devguide.md

```diff
@@ -59,6 +59,17 @@ You can kick off an overall build from the top-level makefile:
 make
 ```
 
+## Adding Experimental Fields
+
+Starting with v0.5.0, all additions to the API must start in the experimental
+release channel. Experimental fields must be marked with the
+`<gateway:experimental>` annotation in Go type definitions. Gateway API CRD
+generation will exclude these fields from stable CRDs.
+
+If experimental fields are removed or renamed, the original field name should be
+removed from the go struct, with a tombstone comment ensuring the field name
```

Comment:
give example of tombstone comment format

Reply:
Added, thanks!